template/cordova/lib/log.js

```diff
@@ -47,10 +47,10 @@ var appTracingInitialState = null,
  * and prints them to console
  */
 module.exports.run = function(args) {
-    var startTime = new Date(new Date().getTime() - 10 * 60 * 1000).toISOString(), // show last 10 minutes by default
-        knownOpts = { 'minutes' : Number, 'dump' : Boolean, 'help' : Boolean },
-        shortHands = { 'mins' : ['--minutes'], 'h' : ['--help'] },
-        parsedOpts = nopt(knownOpts, shortHands, args, 0);
+    var logFromTime = 10 * 60 * 1000; // show last 10 minutes by default
```

Comment:
Might be useful to define `ONE_MINUTE` and then just say `10 * ONE_MINUTE` or `parsedOpts.minutes * ONE_MINUTE`, in order to avoid duplication of `60 * 1000`.